src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -34,6 +34,7 @@
     protected final ChatInterface      chatInterface;
 
     protected final Set<CommandHandle> commands = new HashSet<>();
+    protected final Set<CommandHandle> listeners = new HashSet<>();
```

Comment:
It might be interesting to get oakBot a completely separate bot, given how he internally works different...